scifight/views.py

```diff
@@ -1,3 +1,66 @@
 from django.shortcuts import render
+from django.http import HttpResponse
 
-# Create your views here.
+
+def index(request):
+    return HttpResponse('This is index page')
+
+
+def tournament(request, tournament_slug):
+    return HttpResponse('This is tournament page')
+
+
+def schedule(request, tournament_slug):
+    return HttpResponse('This is schedule page')
+
+
+def fight(request, tournament_slug, fight_id):
+    return HttpResponse('This is fight  #%s page' % fight_id)
+
+
+def rooms(request, tournament_slug):
+    return HttpResponse('This is rooms page')
+
+
+def room(request, tournament_slug, room_id):
+    return HttpResponse('This is room  #%s page' % room_id)
+
+
+def teams(request, tournament_slug):
+    return HttpResponse('This is teams page')
+
+
+def team(request, tournament_slug, team_id_or_slug):
+    return HttpResponse('This is team  #%s page' % team_id_or_slug)
+
+
+def participants(request, tournament_slug):
+    return HttpResponse('This is participants page')
+
+
+def participant(request, tournament_slug, part_id):
+    return HttpResponse('This is participant  #%s page' % part_id)
```

Comment:
«Part» is a bad shorthand for «participant», it's better to write full word here.